src/main/resources/api/swagger.yaml

```diff
@@ -222,6 +218,15 @@ definitions:
             $ref: '#/definitions/Link'
           next:
             $ref: '#/definitions/Link'
+  Review:
+    type: object
+    properties:
+      review-status:
+        type: string
+        description: Status of the Review.
```

Comment:
I think review can also be lowercase case (we are referring the entities on their descriptions as lowercases).